packages/client/components/NewGitLabIssueInput.tsx

```diff
@@ -0,0 +1,264 @@
+  flexDirection: 'column',
+  width: '100%'
+})
+
+const Item = styled('div')({
+  backgroundColor: PALETTE.SLATE_100,
+  cursor: 'pointer',
+  display: 'flex',
+  paddingLeft: 16,
+  paddingTop: 8,
+  paddingBottom: 8
+})
+
+const Issue = styled('div')({
+  display: 'flex',
+  flexDirection: 'column',
+  paddingLeft: 16,
+  width: '100%'
+})
+
+const NewIssueInput = styled('input')({
+  appearance: 'none',
+  background: 'transparent',
+  border: 'none',
+  color: PALETTE.SLATE_700,
+  fontSize: 16,
+  margin: 0,
+  padding: '0px 8px 0px 0px',
+  outline: 0,
+  width: '100%'
+})
+
+const Error = styled(StyledError)({
+  fontSize: 13,
+  textAlign: 'left',
+  width: '100%'
+})
+
+interface Props {
+  isEditing: boolean
+  meetingId: string
+  setIsEditing: (isEditing: boolean) => void
+  viewerRef: NewGitLabIssueInput_viewer$key
+}
+
+const validateIssue = (issue: string) => {
+  return new Legitity(issue).trim().min(2, `C’mon, you call that an issue?`)
+}
+
+const NewGitLabIssueInput = (props: Props) => {
+  const {isEditing, meetingId, setIsEditing, viewerRef} = props
+  const viewer = useFragment(
+    graphql`
+      fragment NewGitLabIssueInput_viewer on User {
+        id
+        team(teamId: $teamId) {
+          id
+        }
+        teamMember(teamId: $teamId) {
+          id
+          integrations {
+            gitlab {
+              api {
+                errors {
+                  message
+                  locations {
+                    line
+                    column
+                  }
+                  path
+                }
+                query {
+                  # use alias otherwise projects args must be the same as GitLabScopingSearchResults query
+                  allProjects: projects(
+                    membership: true
+                    first: 100
+                    sort: "latest_activity_desc"
+                  ) {
+                    edges {
+                      node {
+                        ... on _xGitLabProject {
+                          id
+                          fullPath
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    `,
+    viewerRef
+  )
+  const {id: userId, team, teamMember} = viewer
+  const {id: teamId} = team!
+  const nullableEdges = teamMember?.integrations?.gitlab?.api?.query?.allProjects?.edges ?? []
+  const gitlabProjects = getNonNullEdges(nullableEdges).map(({node}) => node)
+  const atmosphere = useAtmosphere()
+  const {onCompleted, onError} = useMutationProps()
+  const [selectedFullPath, setSelectedFullPath] = useState(gitlabProjects[0]?.fullPath || '')
+  const {fields, onChange, validateField, setDirtyField} = useForm({
+    newIssue: {
+      getDefault: () => '',
+      validate: validateIssue
+    }
+  })
+  const {originRef, menuPortal, menuProps, togglePortal, portalStatus} = useMenu(
+    MenuPosition.UPPER_RIGHT
```

Comment:
+1 can you make this a dropdown? eg `useMenu(x, {isDropdown: true})` and update the github menu to use this, too.

dropdowns start with the full width, whereas menus grow from 0 width, nothing major, just looks nice for wider menus

Reply:
Ooh `isDropdown` looks good! I've added it to Jira as well as GH & GitLab